### PR TITLE
Fix overindented lines.

### PR DESCRIPTION
**Pull request by anntzer**
This is basically pycodestyle's E127 check, except that pycodestyle is
incorrectly too strict in a few places so we can't enforce it
automatically.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by anntzer** [comment]
I have already two PRs open against pycodestyle to fix that.

File: lib/matplotlib/tests/test_colorbar.py [hunk 1]
```diff
@@ -282,27 +282,22 @@ def test_colorbar_minorticks_on_off():
...
-        correct_minorticklocs = np.array([-1.2, -1.1, -0.9, -0.8, -0.7, -0.6,
-                                          -0.4, -0.3, -0.2, -0.1,  0.1, 0.2,
-                                           0.3,  0.4,  0.6,  0.7,  0.8,  0.9,
-                                           1.1,  1.2])
-        np.testing.assert_almost_equal(cbar.ax.yaxis.get_minorticklocs(),
-                                       correct_minorticklocs)
+        np.testing.assert_almost_equal(
+            cbar.ax.yaxis.get_minorticklocs(),
+            [-1.2, -1.1, -0.9, -0.8, -0.7, -0.6, -0.4, -0.3, -0.2, -0.1,
+             0.1, 0.2, 0.3,  0.4,  0.6,  0.7,  0.8,  0.9, 1.1,  1.2])
```
Comment (QuLogic) [inline lib/matplotlib/tests/test_colorbar.py]: A few spaces after comma here that I think are unintentional.

File: lib/matplotlib/tests/test_constrainedlayout.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def test_constrained_layout6():
 def test_constrained_layout7():
     """Test for proper warning if fig not set in GridSpec"""
     with pytest.warns(UserWarning, match='Calling figure.constrained_layout, '
-                            'but figure not setup to do constrained layout'):
+                      'but figure not setup to do constrained layout'):
```
Comment (QuLogic) [inline lib/matplotlib/tests/test_constrainedlayout.py]: This is a bit weird not lining up with `match`.

**Comment by anntzer** [comment]
both handled, thanks for the review

**Pull request merged**